torch/backends/cudnn/__init__.py

```diff
@@ -51,7 +54,7 @@ def _libcudnn():
             lib = find_cudnn_windows_lib()
         else:
             lib = ctypes.cdll.LoadLibrary(None)
-        if hasattr(lib, 'cudnnGetErrorString'):
+        if cuda is not None and hasattr(lib, 'cudnnGetErrorString'):
```

Comment:
I think the `cuda` check here is not needed either.

Reply:
Removed